### PR TITLE
Update discord.ts

### PR DESCRIPTION
**Pull request by Suyashtnt**
very minor

**Please describe the changes this PR makes and why it should be merged:**
Update a comment in `discord.ts`
**Status**

- [x] Code changes have been tested against the Discord API, or there are no code changes

**Semantic versioning classification:**

- [ ] This PR changes the library's interface (methods or parameters added)
  - [x] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [] This PR **only** includes non-code changes, like changes to documentation, README, etc.(I guess it counts?)

### CODE REVIEW

**Comment by Skillz4Killz** [comment]
Hmmm, i think we could possibly just remove this entirely, the API version being changed to used new apis will no longer work. Discord is stopping that and we will not be able to use v9 before its officially released. I have to think about this

**Comment by Skillz4Killz** [comment]
@Suyashtnt I've rebased this to the `next` branch so it can include breaking changes which we merge in the next major release. I believe the best way to handle this is to just remove this entire variable. Discord is no longer going to allow us to customize which version we can use. After the v7 headaches that other libs caused Discord is preventing libraries from using other versions that are not released which makes this variable pretty useless.

1. Remove the API_VERSION variable
2. Hardcode the v8 into the URL

**Comment by Suyashtnt** [comment]
Done, changed accoding to that

File: src/constants/discord.ts [hunk 1]
```diff
@@ -1,15 +1,8 @@
-let API_VERSION = "v8";
-
 export const baseEndpoints = {
-  /** Although, the version can be defaulted, keep the v6 as it can be changed to test newer versions when necessary. */
-  BASE_URL: `https://discord.com/api/${API_VERSION}`,
+  BASE_URL: `https://discord.com/api/V8`,
```
Comment (Skillz4Killz) [inline src/constants/discord.ts:2]:
```suggestion
  BASE_URL: `https://discord.com/api/v8`,
```

**Review by Skillz4Killz — APPROVED** [review]

**Pull request merged**